Validator/Constraints/IsTrueValidator.php

```diff
@@ -83,36 +98,30 @@ public function validate($value, Constraint $constraint)
       *
       * @param string $privateKey
       * @param string $remoteip
-      * @param string $response
+      * @param string $answer
```

Comment:
I renamed $response to $answer since it was not directly clear to me that $reponse ment response from the user, not response from the validation call.